frontend/src/metabase/ui/components/typography/Anchor/Anchor.stories.mdx

```diff
@@ -0,0 +1,81 @@
+import { Canvas, Story, Meta } from "@storybook/addon-docs";
+import { Anchor } from "./";
+import { Text } from "../Text"
+
+<Meta title="Anchor" component={Anchor} />
+
+export const Template = (args) => <Anchor href="https://www.youtube.com/@metabasedata" {...args} />;
+
+# Anchor
+
+Our themed wrapper around [Mantine Anchor](https://mantine.dev/core/anchor/).
+
+## When to use Anchor
+
+The Anchor component allows users to display links with themed styles, and replaces the usage of `<a href="<url>">text</a>`. This component uses the same props as the `Text` component, so it can handle sizing, line clamps, text decoration, and font weight. For regular text, use the `Text` component, and for code, use the `Code` component.
+
+## Docs
+
+- [Figma File](https://www.figma.com/file/h6aMN8H67eu2w8wmDngfnM/Foundation-%2F-text?type=design&node-id=5-70&mode=design&t=9Sh3xAM7HsIZIN1z-11)
+- [Mantine Anchor Docs](https://mantine.dev/core/anchor/)
+
+## Caveats
+
+N/A
+
+## Usage guidelines
+
+N/A
+
+## General
+
+<Canvas>
+  <Story name="Default" args={{
+    children: "You should subscribe to our Youtube channel!",
+    underline: false,
+    italic: false,
+    strikethrough: false,
+    weight: "normal",
+    lineClamp: -1,
+    align: "unset",
+  }}
+  argTypes={
```

Comment:
I'm not seeing any controls show up in Storybook for this component for some reason. I ran `yarn run storybook` locally and all I see is the element on the canvas, but no options to fiddle with the props.